week6.2-ex3.py

```diff
@@ -0,0 +1,21 @@
+from time import time
+
+
+def timer(f, *parameters: any, **parameters2: any) -> float:
+    """
+    The function receives a function and parameters and calculates how long it takes to perform
```

Comment:
I think that execute would be a better term rather than perform.
Also, "the obtained function" - but here I believe too that you can find a better word than obtain.